### PR TITLE
chore: add mask icon

### PR DESCRIPTION
**Pull request by faraaz-deepsource**
Optmized with [SVGOMG](https://jakearchibald.github.io/svgomg/)

### CODE REVIEW

File: src/icons/mask.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="lucide lucide-venetian-mask"><path d="M2 12a5 5 0 0 0 5 5 8 8 0 0 1 5 2 8 8 0 0 1 5-2 5 5 0 0 0 5-5V7h-5a8 8 0 0 0-5 2 8 8 0 0 0-5-2H2Z"/><path d="M6 11c1.5 0 3 .5 3 2-2 0-3 0-3-2ZM18 11c-1.5 0-3 .5-3 2 2 0 3 0 3-2Z"/></svg>
```
Comment (james-deepsource) [inline src/icons/mask.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="lucide lucide-venetian-mask"><path d="M2 12a5 5 0 0 0 5 5 8 8 0 0 1 5 2 8 8 0 0 1 5-2 5 5 0 0 0 5-5V7h-5a8 8 0 0 0-5 2 8 8 0 0 0-5-2H2Z" stroke="currentColor" stroke-width="2"/><path d="M6 11c1.5 0 3 .5 3 2-2 0-3 0-3-2ZM18 11c-1.5 0-3 .5-3 2 2 0 3 0 3-2Z" stroke="currentColor" stroke-width="2"/></svg>
```

**Review by james-deepsource — CHANGES_REQUESTED** [review]

File: src/icons/mask.svg (hunk 1 above)
Comment (james-deepsource) [reply src/icons/mask.svg:1]: Confirm if the stroke width is required once.

File: src/icons/mask.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="lucide lucide-venetian-mask"><path d="M2 12a5 5 0 0 0 5 5 8 8 0 0 1 5 2 8 8 0 0 1 5-2 5 5 0 0 0 5-5V7h-5a8 8 0 0 0-5 2 8 8 0 0 0-5-2H2Z" stroke="currentColor" stroke-width="2"/><path d="M6 11c1.5 0 3 .5 3 2-2 0-3 0-3-2ZM18 11c-1.5 0-3 .5-3 2 2 0 3 0 3-2Z" stroke="currentColor" stroke-width="2"/></svg>
```
Comment (james-deepsource) [inline src/icons/mask.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="lucide lucide-venetian-mask"><path d="M2 12a5 5 0 0 0 5 5 8 8 0 0 1 5 2 8 8 0 0 1 5-2 5 5 0 0 0 5-5V7h-5a8 8 0 0 0-5 2 8 8 0 0 0-5-2H2Z" stroke="currentColor" stroke-width="2"/><path d="M6 11c1.5 0 3 .5 3 2-2 0-3 0-3-2ZM18 11c-1.5 0-3 .5-3 2 2 0 3 0 3-2Z" stroke="currentColor"/></svg>
```

**Review by james-deepsource — CHANGES_REQUESTED** [review]
Let's get rid of the `stroke-width` attributes; it appears a bit thicker. Also, please ensure to run `yarn build-lib-icons` and commit the changes.

**Review by shuvam-deepsource — APPROVED** [review]

**Review by james-deepsource — APPROVED** [review]

**Pull request merged**